iOS/rainbow/Controller/Booklet/BookletInstructionController.swift

```diff
@@ -88,13 +88,6 @@ class BookletInstructionController: BookletBaseController {
     // MARK: - View Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
-                contentImageView!.image = image
-        //        pageTitleView!.text = titleString
-        //        subtitleView!.text = subTitleString
-        //        statement?.text = statementString
-        //        link = linkString
-        //
-        //        button?.layer.cornerRadius = 20
-        //        button?.clipsToBounds = true
+        contentImageView!.image = image
```

Comment:
I'll loop around and clean this up, but let's always be wary of force-unwrapping optionals in code. This is the main cause of crashes in Swift apps.